docs/6-the-headliner/2-dark-launches.md

```diff
@@ -0,0 +1 @@
+# Dark Launches
```

Comment:
@ckavili will work with KServe? I've found this https://github.com/kserve/kserve/issues/1672